client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -49,7 +49,7 @@
 		cursor: pointer;
 		float: left;
 		width: 100%;
-		margin: 0;
+		margin: 32px; // only applies in IE
```

Comment:
I've observed this, but why?

Reply:
Because of this https://github.com/Automattic/wp-calypso/pull/42570/files/8d5ab7a387671546fe14370ae2074cf8af536393#diff-f33d6a8ea49174076a1f441a7b951eb1R79